openlibrary/macros/BookByline.html

```diff
@@ -1,21 +1,26 @@
-$def with(author_names_and_urls, limit=None, overflow_url=None, attrs='')
+$def with(author_names_url_birth_date_death_date, limit=None, overflow_url=None, attrs='')
```

Comment:
This should only _optionally_ display the birth/death; e.g. `show_years=True/False`.

(Note: Because we only want to include the years on books pages, the librarian-check should happen _there_, not in this template).